### ISSUE TITLE
FB OpenID login

### ISSUE THREAD

**Issue by jdaudier** [loicginoux/joanne_project#132]
Eris suggested:

Why do I need to create an account when I sign-in via FB?

To sign up.... we do

1) http://www.foodrubix.com/register
2) click Login with Facebook
3) walk through the instructions on allowing FB apps with permissions
4) should direct to http://www.foodrubix.com/<<user>>

but now... the flow is
1 ~ 3 the same
4) show 'create account' page (with most of the things filled in except the password
5) http://www.foodrubix.com/<<user>>

basically, I would thought that if I choose 'Login with Facebook', it does something what we called 'OpenID' and does not need to create a password for this foodrubix site, one less password to type/remember (even though they might be the same)

**Comment by loicginoux** [comment]
This is not needed but I thought that would be needed for being able to sign in using the normal way. I thought about it as well when I implemented it but thought that if people registered with Facebook and then later on wanted to log in using the normal way (username + password) then they couldn't because they wouldn't have a password registered in the site. 
I could just skip this extra step but then people will always have to log in using Facebook (or use the recover password page). If this is fine with you then I will remove that extra step.

**Comment by jdaudier** [comment]
So right now they can sign up with FB and also log in?  Do you think it's better to have it both ways?

**Comment by loicginoux** [comment]
Yes right now they can log in with both methods. I thought it was better to have it both ways when I implemented it, but it's true that it is a bit redundant.

**Comment by jdaudier** [comment]
OK - let's leave it as is and revisit later to see if we think we should change it
